README.md

```diff
@@ -1 +1,46 @@
-# Portfolio-project
\ No newline at end of file
+
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
```

Comment:
## [OPTIONAL]
- [ ] I think it would be nice to add a screenshot of your home section in your `README`